workflows/alphafold2.nf

```diff
@@ -72,17 +72,29 @@ workflow ALPHAFOLD2 {
     //
     // SUBWORKFLOW: Read in samplesheet, validate and stage input files
     //
-    INPUT_CHECK (
-        ch_input
-    )
-    .reads
-    .map {
-        meta, fasta ->
-
-        [ meta, fasta.splitFasta(file:true) ]
-    }
-    .transpose()
+    if (params.model_preset != 'multimer') {
+        INPUT_CHECK (
+            ch_input
+        )
+        .reads
+        .map {
+            meta, fasta ->
+            [ meta, fasta.splitFasta(file:true) ]
+        }
+        .transpose()
     .set { ch_fasta }
+    } else {
+        INPUT_CHECK (
+            ch_input
+        )
+        .reads
+        .map {
+            meta, fasta ->
+            [ meta, fasta ]
+        }
+        .transpose()
```

Comment:
Do we need this here? Is not returning just the same structure

Reply:
Yes, you are right! Fixed in the next commits